FLEDGE.md

```diff
@@ -1146,16 +1146,36 @@ Note that the key fields are used by the browser both to verify the signature, a
 
 The browser ensures, using TLS, the authenticity and integrity of information provided to the auction through calls made directly to an ad tech's servers. This guarantee is not provided for data passed in `runAdAuction()`. To account for this, additional bids use the same HTTP response header interception mechanism that's already in use for the [Bidding & Auction response blob](FLEDGE_browser_bidding_and_auction_API.md#step-3-get-response-blobs-to-browser) and `directFromSellerSignals`.
 
-To use HTTP response headers to convey the additional bids, the request to fetch them will first need to specify the `adAuctionHeaders` Fetch flag or `adAuctionHeaders` attribute on an iframe element. The procedure for specifying `adAuctionHeaders` on a Fetch or an iframe navigation is described in section [2.5.2 Using Response Headers](#252-using-response-headers), except that the response would use `Ad-Auction-Additional-Bid` response headers instead of `Ad-Auction-Signals`, and the caller of `runAdAuction()` would specify `additionalBids` on the auction config instead of or in addition to `directFromSellerSignalsHeaderAdSlot`.
+ `Ad-Auction-Signals` response header 
 
-Using `adAuctionHeaders` will cause the Fetch or navigation request to include an additional request header, `Sec-Ad-Auction-Fetch: ?1`. This also signals to the browser that it should look for one or more additional bids encoded as HTTP response headers from this Fetch or navigation. Each instance of the `Ad-Auction-Additional-Bid` response header will correspond to a single additional bid. The response may include more than one additional bid by specifying multiple instances of the `Ad-Auction-Additional-Bid` response header. The structure of each instance of the `Ad-Auction-Additional-Bid` header must be as follows:
+Servers return additional bids to the browser using the `Ad-Auction-Additional-Bid` response header of some `fetch()` request or `iframe` navigation, together with the `additionalBids` parameter on `navigator.runAdAuction()`. This uses the same syntax as that used to convey `directFromSellerSignals` [using response headers](#252-using-response-headers).
```

Comment:
>the `additionalBids` parameter on `navigator.runAdAuction()`.

Use of parameter caused me to pause and search - making me think there was a method parameter. 
Suggest perhaps _field_ as described elsewhere.

Reply:
That makes sense. I've replaced it here and in two other places where it made sense to do so. Thanks.